sdks/python/apache_beam/typehints/typehints_test.py

```diff
@@ -320,6 +320,14 @@ def test_getitem_proxy_to_union(self):
     hint = typehints.Optional[int]
     self.assertTrue(isinstance(hint, typehints.UnionHint.UnionConstraint))
 
+  def test_is_optional(self):
```

Comment:
Add more tests to check encoding/decoding with NullableCoder ? (we may already have such tests)

Reply:
We are, Brian wrote them when NullableCoder was just used for nested encoding